### ISSUE TITLE
Duda atributo por referencia

### ISSUE THREAD

**Issue by juanfdezg** [RodrigoToroIcarte/IIC2113-2023-1#200]
Hola! Me vino una sobre Clean Code. Supongamos que tengo una clase A y una clase B, si es que paso como atributo una instancia de la clase A a un método de la clase B, y este método realiza cambios sobre los atributos de la clase A, no estaría rompiendo ninguna regla de Clean Code cierto? Adjunto foto del ejemplo para que se entienda mejor.

![image](https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/89492735/5cf9afb4-9b1b-4875-af40-280ba4418e51)

**Comment by RodrigoToroIcarte** [comment]
No, en el sentido de que la ClaseA es una estructura de datos. Su objetivo es agrupar y mostrar datos.
